### PR TITLE
Edit suggested in file admin_guide/scheduling/pod_placement.adoc

### PR DESCRIPTION
**Pull request by mburke5678**
https://github.com/openshift/openshift-docs/issues/9257

### CODE REVIEW

File: admin_guide/scheduling/pod_placement.adoc [hunk 1]
```diff
@@ -293,26 +293,29 @@ endif::[]
...
+    scheduler.alpha.kubernetes.io/node-selector: env=test,infra=fedora <1>
+spec: {},
+status: {}
 ----
 +
 <1> Annotation to create the labels to match the project label selector. Here, the key/value labels are `env=test` and `infra=fedora`.
++
+[NOTE]
+====
+When using the *Pod Node Selector* admission controller, you cannot use the `oc new-project` command as
```
Comment (aveshagarwal) [inline admin_guide/scheduling/pod_placement.adoc]: Not `oc new-project`, but oc adm your-project for setting project node selector.

File: admin_guide/scheduling/pod_placement.adoc [hunk 2]
```diff
@@ -293,26 +293,29 @@ endif::[]
...
+that command uses a different admission controller.
```
Comment (aveshagarwal) [inline admin_guide/scheduling/pod_placement.adoc]: mentioning about NodeEnv would be helpful i think.

File: admin_guide/scheduling/pod_placement.adoc [hunk 3]
```diff
@@ -293,26 +293,29 @@ endif::[]
...
+spec: {},
+status: {}
 ----
 +
 <1> Annotation to create the labels to match the project label selector. Here, the key/value labels are `env=test` and `infra=fedora`.
++
+[NOTE]
+====
+When using the *Pod Node Selector* admission controller, you cannot use `oc adm new-project <project-name>` for setting project node selector. 
+The `oc adm new-project` command uses the `NodeEnv` admission controller.
```
Comment (aveshagarwal) [inline admin_guide/scheduling/pod_placement.adoc]: Sorry to be nit-picking here. I'd recommend something like (words you can change whatever way you want): when setting project node selectors via `oc adm new-project myproject --node-selector='type=user-node,region=east`, it sets following annotation: `openshift.io/node-selector` and this annotation is processed/operated/used by NodeEnv admission plugin. Just trying to be as clear as possible.

**Comment by mburke5678** [comment]
@mdshuai Can someone on your team review this PR?

**Comment by mburke5678** [comment]
@mdshuai Can someone on your team review this PR?

cc @xltian

**Comment by mdshuai** [comment]
LGTM, thanks

**Comment by mburke5678** [comment]
@openshift/team-documentation PTAL

File: admin_guide/scheduling/pod_placement.adoc [hunk 4]
```diff
@@ -293,26 +293,30 @@ endif::[]
...
+spec: {},
+status: {}
 ----
 +
 <1> Annotation to create the labels to match the project label selector. Here, the key/value labels are `env=test` and `infra=fedora`.
++
+[NOTE]
+====
+When using the *Pod Node Selector* admission controller, you cannot use `oc adm new-project <project-name>` for setting project node selector. 
+Setting the project node selector using the `oc adm new-project myproject --node-selector='type=user-node,region=<region>` {product-title} sets
```
Comment (kalexand-rh) [inline admin_guide/scheduling/pod_placement.adoc]: maybe s/Setting the project node selector using the *command* {product-title} sets/When you set the project node selector by using the *command* command, {product-title} sets

**Comment by mburke5678** [comment]
@aveshagarwal What OpenShift versions does this change affect?

**Comment by mburke5678** [comment]
Per @aveshagarwal this should roll back to 3.5. 
Scheduler was a single topic in 3.5, broken out into subsections for 3.6+. Will change 3.5 in separate PR.

**Pull request merged**